src/adagrams.js

```diff
@@ -1,6 +1,151 @@
+const POOL = {
+  a: 9,
+  b: 2,
+  c: 2,
+  d: 4,
+  e: 12,
+  g: 3,
+  h: 2,
+  i: 9,
+  j: 1,
+  k: 1,
+  l: 4,
+  m: 2,
+  n: 6,
+  o: 8,
+  p: 2,
+  q: 1,
+  r: 6,
+  s: 4,
+  t: 6,
+  u: 4,
+  v: 2,
+  w: 2,
+  x: 1,
+  y: 2,
+  z: 1,
+}
+
 const Adagrams = {
+  randInt(max) {
+    return Math.floor(Math.random() * Math.floor(max))
+  },
+
   drawLetters() {
-    // Implement this method for wave 1
+    // Turn pool object into flat array
+    let newPool = [];
+    for (var letter in POOL) {
```

Comment:
oops, oops! no `var`!

Reply:
This is what I get for looking up syntax on StackOverflow and mindlessly typing 😂